lib/chai/config.js

```diff
@@ -50,6 +50,21 @@ module.exports = {
    * @api public
    */
 
-  truncateThreshold: 40
+  truncateThreshold: 40,
 
+  /**
+   * ### config.useProxy
+   *
+   * User configurable property, define if chai will use a Proxy to throw
```

Comment:
**Nitpicky Warning:** The word `define` here should be written as `defines`, shouldn't it?
(Please correct me if I'm wrong, I'm not a native speaker)